examples/tasks_tutorial.py

```diff
@@ -142,3 +142,69 @@
 ids = [2, 1891, 31, 9983]
 tasks = openml.tasks.get_tasks(ids)
 pprint(tasks[0])
+
+############################################################################
+# Creating tasks
+# ^^^^^^^^^^^^^^^^^
+#
+# You can also create new tasks. Take the following into account:
+#
+# * You can only create tasks on _active_ datasets
+# * For now, only the following tasks are supported: classification, regression,<br>
+# clustering, and learning curve analysis.
+# * For now, tasks can only be created on a single dataset.
+# * The exact same task should not already exist.
+#
+# Creating a task requires the following input:
+#
+# * task_type_id: The task type ID, required (see below). Required.
+# * dataset_id: The dataset ID. Required.
+# * target_name: For supervised tasks, the name of the attribute you aim to predict.<br>
+# Optional.
+# * estimation_procedure_id : The ID of the estimation procedure used to create train-test<br>
+# splits. Optional.
+# * evaluation_measure: The name of the evaluation measure. Optional.
+# * Any additional inputs for specific tasks
+#
+# It is best to leave the evaluation measure open if there is no strong prerequisite for a
+# specific measure. OpenML will always compute all appropriate measures and you can filter
+# or sort results on your favourite measure afterwards. Only add an evaluation measure if
+# necessary (e.g. when other measure make no sense), since it will create a new task, which
+# scatters results across tasks.
+
+
+############################################################################
+# Example
+# #######
+#
+# Let's create a classification task on a new dataset. We first upload the dataset
+# (just a copy of the Iris dataset in this example) and then create a new classification
+# task on it. We'll use 10-fold cross-validation (ID=1), without a predefined measure.
+
+dataset = openml.OpenMLDataset(
+    "TaskCreationTestDataset",
+    "test",
+    data_format="arff",
+    url="https://www.openml.org/data/download/61/dataset_61_iris.arff",
+)
+dataset.publish()
+openml.datasets.status_update(dataset.dataset_id, "active")
+
+tasktypes = openml.tasks.TaskTypeEnum
+my_task = openml.tasks.create_task(task_type_id=tasktypes.SUPERVISED_CLASSIFICATION,
```

Comment:
Could you please format this function call the same way you formatted the creation of the dataset above?